winui/Ribbon/RibbonQAT.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: Quick Access ToolBar | WinUI | Ribbon | Syncfusion
+description: Learn more about the Quick Access ToolBar  in the Syncfusion WinUI Ribbon (SfRibbon) control.
+platform: winui
+control: Ribbon
+documentation: ug
+---
+
+# Quick Access ToolBar in WinUI Ribbon
+
+The 'Quick Access Toolbar' is used to group frequently used commands above or below the Ribbon, and it allows users to add or remove commands to/from it. This feature provides end users with easy accessibility.

```

Comment:
The quick access toolbar (QAT) is used to render a set of ribbon items that are commonly used in applications. It renders at the top-left corner of a window or ribbon to make it more accessible. Users can place it above or below the ribbon, remove commands from it, or add commands to it.